### PR TITLE
[#21] display version in admin

### PR DESCRIPTION
**Pull request by SonnyBA**
Partially fixes https://github.com/maykinmedia/open-api-framework/issues/21 : 

image

### CODE REVIEW

**Review by Coperh — COMMENTED** [review]
Similar situation as mentioned in this comment: https://github.com/open-zaak/open-notificaties/pull/155#discussion_r1630819492

**Comment by annashamray** [comment]
Please rebase on master, quick-start action is fixed there

File: src/objecttypes/conf/base.py
Comment (stevenbal) [inline src/objecttypes/conf/base.py:34]: This should be fixed if you rebase on master I think

File: src/objecttypes/conf/base.py
Comment (stevenbal) [inline src/objecttypes/conf/base.py:16]: I think it would be better to add this to `INSTALLED_APPS` in `open_api_framework/conf/base.py` itself :)

File: src/objecttypes/conf/base.py
Comment (stevenbal) [inline src/objecttypes/conf/base.py:48]: I think if you add this line to the `TEMPLATES` in `open_api_framework/conf/base.py` we can avoid this duplicated code

File: src/objecttypes/scss/admin/_admin_theme.scss
Comment (stevenbal) [inline src/objecttypes/scss/admin/_admin_theme.scss:41]: Maybe we should make this a bit darker? It's not very readable imo

image

**Review by stevenbal — CHANGES_REQUESTED** [review]
Looks good, but some minor comments

Also it seems that Objecttypes also shows the version on the dashboard if you're not logged in:

image

Can you remove this?

File: src/objecttypes/scss/admin/_admin_theme.scss
Comment (SonnyBA) [reply src/objecttypes/scss/admin/_admin_theme.scss]: Now that it reuses another variable (same as the welcome message):
image

File: src/objecttypes/scss/admin/_admin_theme.scss
Comment (Coperh) [inline src/objecttypes/scss/admin/_admin_theme.scss:44]: Can these be moved to admin.css in open-api-framework or are they only available here?

Comment (stevenbal) [reply src/objecttypes/scss/admin/_admin_theme.scss:44]: This relies on SCSS variables (`$color-secondary`) which only exist in the project itself, so I don't think we can easily move this to OAf

**Review by stevenbal — APPROVED** [review]
Approving, but let's wait until the PR in OAF is merged and released and then pin it to the pypi release here

File: src/objecttypes/scss/admin/_admin_theme.scss
Comment (SonnyBA) [reply src/objecttypes/scss/admin/_admin_theme.scss:44]: Reasoning behind this was that we reuse the admin theme colors here (of the project itself) instead of possibly introducing new colors through OAF.

**Pull request merged**
